app/kubemci/pkg/gcp/healthcheck/healthchecksyncer.go

```diff
@@ -100,6 +91,37 @@ func (h *HealthCheckSyncer) EnsureHealthCheck(lbName string, ports []ingressbe.S
 	return ensuredHealthChecks, err
 }
 
+func getHealthCheckPath(port ingressbe.ServicePort, clients map[string]kubernetes.Interface) (string, error) {
+	var err error
+	path := ""
+	for cName, c := range clients {
+		probe, prErr := kubeutils.GetProbe(c, port)
+		if prErr != nil {
+			prErr = fmt.Errorf("Error %s when getting probe for port %v", prErr, port)
```

Comment:
"getting probe for port" is not really clear to the user.
How about saying "getting readiness probe from pods for service \<service-name\>?"